src/xenia/vfs/devices/stfs_container_device.cc

```diff
@@ -617,6 +612,20 @@ bool StfsHeader::Read(const uint8_t* p) {
   title_thumbnail_image_size = xe::load_and_swap<uint32_t>(p + 0x1716);
   std::memcpy(thumbnail_image, p + 0x171A, 0x4000);
   std::memcpy(title_thumbnail_image, p + 0x571A, 0x4000);
+
+  // Metadata v2 Fields
+  std::memcpy(series_id, p + 0x3B1, 0x10);
```

Comment:
Should this be checking the metadata version is 2 (or greater) before reading these fields?

Reply:
In the case of a game being Metadata v1, these offsets are unused, so they contain null data. Considering all of the fields added in Metadata v2 are irrelevant to emulation and will default to the same null data in `struct StfsHeader` if not read, I don't think it'll be an issue, but can be corrected if it's considered a code quality issue.

Reply:
Conditionally parsing these fields as of the latest commit.